### PR TITLE
correct precision in the no config file message

### LINKED ISSUE #7834: empty config file is reported as no config file `
empty config file...
```
[root@localhost vagrant]# cat /openshift.local.config/master/admin.kubeconfig
[root@localhost vagrant]# oc get nodes --config=/openshift.local.config/master/admin.kubeconfig
error: No configuration file found, please login or point to an existing file:
```
sais no config file
```
error: No configuration file found, please login or point to an existing file:
  1. Via the command-line flag --config
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
```

### PR DESCRIPTION
**Pull request by jayunit100**
fixes #7834

### CODE REVIEW

File: pkg/cmd/util/clientcmd/factory.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ func (c defaultingClientConfig) ClientConfig() (*kclient.Config, error) {
 		return icc, nil
 	}
 
-	return nil, fmt.Errorf(`No configuration file found, please login or point to an existing file:
+	return nil, fmt.Errorf(`No (or empty) configuration file found, please login or point to an existing file:
```
Comment (liggitt) [inline pkg/cmd/util/clientcmd/factory.go]: that reads very strangely. pretty sure there are tests that need updating as well

**Comment by jayunit100** [comment]
@liggitt message looks better now ? (also updated the test)

**Comment by liggitt** [comment]
@smarterclayton wdyt?

**Comment by smarterclayton** [comment]
Looks fine to me.

**Comment by jayunit100** [comment]
should we push this in? its trivial but i think will help some people (like me:))  or i can close..

**Comment by danmcp** [comment]
[merge]

**Pull request merged**